framework/src/reporters/ParsedScalarReporter.C

```diff
@@ -0,0 +1,57 @@
+//* This file is part of the MOOSE framework
+//* https://mooseframework.inl.gov
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "ParsedScalarReporter.h"
+
+registerMooseObject("MooseApp", ParsedScalarReporter);
+
+InputParameters
+ParsedScalarReporter::validParams()
+{
+  InputParameters params = ParsedReporterBase::validParams();
+  params.addRequiredParam<std::vector<ReporterName>>("scalar_reporter_names",
+                                                     "Scalar reporter names to apply function to.");
+  params.set<std::vector<std::string>>("reporter_symbols") = {};
+  params.suppressParameter<std::vector<std::string>>("reporter_symbols");
```

Comment:
I think you should just rename the reporter_symbols parameter?

Reply:
I already have the scalar_reporter_symbols from the base class so I use that with scalar_reporter_names.  That follows the pattern I use in ParsedVectorReporter.